README.md

```diff
@@ -87,6 +87,7 @@ of randomness based on their specific needs:
 | `wasm_js`         | Web Browser, Node.js | `wasm32‑unknown‑unknown`, `wasm32v1-none` | [`Crypto.getRandomValues`]. Requires feature `wasm_js` ([see below](#webassembly-support)).
 | `efi_rng`         | UEFI                 | `*-unknown‑uefi`         | [`EFI_RNG_PROTOCOL`] with `EFI_RNG_ALGORITHM_RAW` (requires `std` and Nigthly compiler)
 | `custom`          | All targets          | `*`                      | User-provided custom implementation (see [custom backend])
+| `runtime_error`   | All targets          | `*`                      | Errors when providing randomness. Useful when only needing this crate to compile in wasm32, but it's not actually used.
```

Comment:
I would write the description a bit shorter:
>Always returns `Err(Error::UNSUPPORTED)` (see [unsupported backend](link))

You can remove the `Error::UNSUPPORTED` example from the custom backend section and add a new section for this backend with description of when it should be used.

Reply:
I updated it. There was actually an example in the "custom backend" section about creating a custom backend that returns Unsupported, but it's not an ideal recommendation because it creates a Wasm module import in the case of compiling for Wasm. I moved that paragraph into the new "Unsupported backend" section